core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -484,6 +484,45 @@ void core_init_mmu_prtn(struct mmu_partition *prtn, struct tee_mmap_region *mm)
 	}
 }
 
+/*
+ * In order to support 32-bit TAs we will have to find
+ * a TAs VA base in the region (0, 4GB).
```

Comment:
` * a user VA base in the region [0, 4GB].`

Reply:
`)` means excluding in this case. Changed to `[1GB, 4GB)`